docs/AboutUs.md

```diff
@@ -5,55 +5,35 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
-
 ## Project team
 
-### John Doe
+### Fong Wei Jie
 
 <img src="images/johndoe.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Project Advisor
+* Role: Code Quality
+* Responsibilities: Looks after code quality, ensures adherence to coding standards, etc.
 
-### Jane Doe
+### Javier Lim Xin Long
 
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Team Lead
-* Responsibilities: UI
-
-### Johnny Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
-
-* Role: Developer
-* Responsibilities: Data
+* Role: Project Manager
+* Responsibilities:
+    * Deliverables and deadlines: Ensure project deliverables are done on time and in the right format. 
+    * Scheduling and tracking: In charge of defining, assigning, and tracking project tasks.
+    
 
-### Jean Doe
 
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+### Heng Jie Kai Joven
 
-* Role: Developer
-* Responsibilities: Dev Ops + Threading
+<img src="images/joven-heng.png" width="200px">
 
-### James Doe
+[Joven github](http://github.com/Joven-Heng) 
 
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```

Comment:
Perhaps we could keep keep this so we can change it in future

Reply:
Ok fixed those, have a look